src/main/java/ch/njol/skript/structures/StructVariables.java

```diff
@@ -146,8 +145,13 @@ public List<NonNullPair<String, Object>> getVariables() {
 	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult, EntryContainer entryContainer) {
 		SectionNode node = entryContainer.getSource();
 		node.convertToEntries(0, "=");
-
-		List<NonNullPair<String, Object>> variables = new ArrayList<>();
+		List<NonNullPair<String, Object>> variables;
+		Script script = getParser().getCurrentScript();
+		DefaultVariables existing = script.getData(DefaultVariables.class); // if the user has TWO variables: sections
+		if (existing != null && existing.hasDefaultVariables())
```

Comment:
Not easy to write a suggestion as I'm on mobile, but should have curly braces for this if-else structure :)